src/2.py

```diff
@@ -0,0 +1,40 @@
+import re
+
+
+print('Hello World')
+
+name = input('What is your name?\n')
+print('Hi, %s ' % name)
+
+friends = ['john', 'pat', 'gary', 'michael']
+for i, name in enumerate(friends):
+    print("iteration {iteration} is {name}".format(iteration=i, name=name))
+
+parents, babies = (1, 1)
+while babies < 100:
+    print('This generation has {0} babies'.format(babies))
+    parents, babies = (babies, parents + babies)
+
+
+def greet(par):
+    print('Hello', par)
+
+
```

Comment:
it looks strange that our greet function separated from it's calls with 2 empty lines and next task with re with only one